hack/verify-gofmt.sh

```diff
@@ -6,32 +6,21 @@ set -o pipefail
 
 GO_VERSION=($(go version))
 
-if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.4|go1.5') ]]; then
-  echo "Unknown go version '${GO_VERSION[2]}', skipping gofmt."
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.[4-6]') ]]; then
+  echo "Unknown go version '${GO_VERSION[@]:2}', skipping gofmt." >&2
   exit 0
 fi
 
 STI_ROOT=$(dirname "${BASH_SOURCE}")/..
-source "${STI_ROOT}/hack/common.sh"
+source "${STI_ROOT}/hack/util.sh"
 
 cd "${STI_ROOT}"
 
-find_files() {
-  find . -not \( \
-      \( \
-        -wholename './output' \
-        -o -wholename './_output' \
-        -o -wholename './release' \
-        -o -wholename './target' \
-        -o -wholename '*/Godeps/*' \
-      \) -prune \
-    \) -name '*.go'
-}
-
-bad_files=$(find_files | xargs gofmt -s -l)
+bad_files=$(s2i::util::find_files | xargs gofmt -s -l)
 if [[ -n "${bad_files}" ]]; then
-  echo "!!! gofmt needs to be run on the following files: "
+  echo "!!! gofmt needs to be run on the following files: " >&2
   echo "${bad_files}"
```

Comment:
See https://github.com/openshift/origin/pull/6820

If it was sent to `stderr` we wouldn't be able to use the `xargs` approach to autocorrect.